src/System.IO.FileSystem/src/System/IO/UnixFileSystem.cs

```diff
@@ -50,8 +50,8 @@ public override void CopyFile(string sourceFullPath, string destFullPath, bool o
 
             // Copy the contents of the file from the source to the destination, creating the destination in the process
             const int bufferSize = 0x1000;
-            using (UnixFileStream src = new UnixFileStream(sourceFullPath, FileMode.Open, FileAccess.Read, FileShare.Read, bufferSize, FileOptions.None))
-            using (UnixFileStream dst = new UnixFileStream(destFullPath, overwrite ? FileMode.CreateNew : FileMode.Create, FileAccess.Read, FileShare.Read, bufferSize, FileOptions.None))
+            using (UnixFileStream src = new UnixFileStream(sourceFullPath, FileMode.Open, FileAccess.Read, FileShare.Read, bufferSize, FileOptions.None, null))
+            using (UnixFileStream dst = new UnixFileStream(destFullPath, overwrite ? FileMode.CreateNew : FileMode.Create, FileAccess.Read, FileShare.Read, bufferSize, FileOptions.None, null))
```

Comment:
@ericstj  can confirm, but I believe this is going to cause the implementation to blow up with null ref exceptions.  I think the right fix is to change this from using UnixFileStream to instead just use FileStream; internally FileStream will create the UnixFileStream with itself as the parent.